src/styles/FunButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ButtonHTMLAttributes, FC } from 'react'
+import styled from 'styled-components'
+
+const BaseButton = styled.button`
+  width: 5.5rem;
+  height: 2rem;
+
+  background-color: #7d53d6;
+  border-radius: 5px;
+  border: none;
+
+  color: white;
+  cursor: pointer;
+
+  :hover {
+    background-color: rgba(125, 83, 214, 0.8);
+  }
+`
+
+const FunButton: FC<ButtonHTMLAttributes<HTMLButtonElement>> = ({
+  children,
+  ...otherProps
+}) => {

```

Comment:
(p3) React 18 부터는 FC 에 children 이 제거되는걸로 알고있는데요, Props 타입을 정의해서 사용해야 할 것 같습니다
+ FC 와 함수를 직접 사용해서 컴포넌트를 만드는게 혼용되고있는데요, 통일되는 편이 좋을 것 같습니다

```suggestion
type OwnProps = {
  children?: React.ReactNode
} & ButtonHTMLAttributes<HTMLButtonElement>

const FunButton = ({
  children,
  ...otherProps
}: OwnProps) => {
```